sympy/parsing/c/c_parser.py

```diff
@@ -684,7 +653,7 @@ def transform_call_expr(self, node):
 
         def transform_return_stmt(self, node):
             """Returns the Return Node for a return statement"""
-            return Return(next(node.get_children()).spelling)
+            return Return(Symbol(next(node.get_children()).spelling))
```

Comment:
I'm not sure if this is the right way to handle this. You don't necessarily always return a Symbol.

Reply:
You're right. I added it to see if it fixes the bug, but (unsurprisingly) it doesn't. I'll remove it, as it was accidentally committed.